packages/instant-apps-components/src/components/instant-apps-app-guide/instant-apps-app-guide.scss

```diff
@@ -0,0 +1,86 @@
+:host {
+  --instant-apps-app-guide-width: 300px;
+  --instant-apps-app-guide-height: 400px;
+  --instant-apps-app-guide-list-item-bubble-size: 22px;
+
+  width: var(--instant-apps-app-guide-width);
+  height: var(--instant-apps-app-guide-height);
+  display: block;
+
+  calcite-panel {
+    max-width: var(--instant-apps-app-guide-width);
+    max-height: var(--instant-apps-app-guide-height);
+    display: flex;
+  }
+
+  calcite-carousel {
+    flex: 1;
+    overflow: scroll;
+    height: fit-content;
+    background-color: var(--calcite-color-foreground-1);
+  }
+
+  [slot='header-content'] {
+    display: flex;
+    gap: var(--calcite-spacing-sm);
+    padding-inline-start: 0.25rem;
+  }
+
+  .content-heading {
+    font-weight: var(--calcite-font-weight-bold);
+    display: flex;
+    gap: var(--calcite-spacing-sm);
+    line-height: var(--calcite-font-line-height-fixed-lg);
+  }
+
+  calcite-carousel-item {
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+  }
+
+  calcite-carousel-item div {
```

Comment:
I think it'd be good to add a more specific css selector here for the page content instead of just `div`